### ISSUE TITLE
Yearly deltaboard missed someone?

### ISSUE THREAD

**Issue by hacksoncode** [MystK/delta-bot-three#173]
Got a message from /u/allsfair86 asking why they aren't on the yearly deltaboards since they've received 12 deltas since January and the lowest entry on the board has only 10. 

Seems like a good question.

**Comment by MystK** [comment]
Hi @RaoulMeyer, could I assign this to you? You probably have the most experience with this.

**Comment by RaoulMeyer** [comment]
Sure, I think I already know what the issue is, but it is going to be hard to find a way to fix it unfortunately.

**Comment by Snorrrlax** [comment]
It would probably be more efficient to focus on the alternative method we talked about before?

**Comment by RaoulMeyer** [comment]
Yeah I forgot for a moment that the missing delta solution would also solve the yearly deltaboard problems.

**Comment by MystK** [comment]
Could you explain a little bit of what's causing it?

**Comment by RaoulMeyer** [comment]
The current problem with the yearly deltaboard is caused by the search not returning deleted threads. Any deltas that were awarded in deleted threads are therefore not found. For most users the yearly deltaboard will be correct, but for some users deltas will be missing.

For allsfair86, the following delta is not found: https://www.reddit.com/r/changemyview/comments/5vvjaa/cmv_english_classes_should_place_much_less/

That still leaves 11 deltas, although the last delta might not have been detected yet at the time this issue was made.

The solution that @Snorrrlax and I talked about can be found in the chat logs on Slack if you're interested.

**Comment by MystK** [comment]
Also looks like comments that are nested are not counted.
